app/src/debug/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,24 @@
+<resources>
+    <!--
+    TODO: Before you run your application, you need a Google Maps API key.
+
+    To get one, follow this link, follow the directions and press "Create" at the end:
+
+    https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=86:13:3C:18:B6:AB:0C:AA:D5:5C:75:6F:49:9D:CE:50:11:92:73:EC%3Bcom.github.sdpteam15.polyevents.fragments
+
+    You can also add your credentials to an existing key, using these values:
+
+    Package name:
+    com.github.sdpteam15.polyevents.fragments
+
+    SHA-1 certificate fingerprint:
+    86:13:3C:18:B6:AB:0C:AA:D5:5C:75:6F:49:9D:CE:50:11:92:73:EC
+
+    Alternatively, follow the directions here:
+    https://developers.google.com/maps/documentation/android/start#get-key
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">AIzaSyAL2oTIMqy1leoIi791kAwjppazruq7nO8</string>
```

Comment:
It's best not to share the api key on github. Maybe we can put this file in git ignore.